src/inverted_index.h

```diff
@@ -63,11 +63,6 @@ typedef union {
   const NumericFilter *filter;
 } IndexDecoderCtx;
 
-/**
- * Called when an entry is removed
- */
-typedef void (*RepairCallback)(const RSIndexResult *res, void *arg);
```

Comment:
:+1: I wonder if there would be a way to automatically figure out all the used C API so we can remove all of it at once and then not have to worry about this in the future. :thinking: